lib/whitehall_importer/integrity_checker.rb

```diff
@@ -135,5 +182,19 @@ def default_image?(proposed_image_payload, publishing_api_image, attribute)
         proposed_image_payload.empty? &&
         publishing_api_image[attribute] == "placeholder"
     end
+
+    def edition_unpublishing_explanation
+      edition.status.details.try(:public_explanation) ||
+        edition.status.details.try(:explanatory_note)
+    end
```

Comment:
I don't think anything uses this method anymore.

Reply:
Good catch - removed